Actions/CheckForUpdates/CheckForUpdates.ps1

```diff
@@ -374,8 +374,8 @@ try {
                 
                 # If $directCommit, then changes are made directly to the default branch
                 if (!$directcommit) {
-                    # If not direct commit, create a new branch with a random name, and switch to it
-                    $branch = [System.IO.Path]::GetRandomFileName()
+                    # If not direct commit, create a new branch with name, relevant to the current date and base branch, and switch to it
+                    $branch = "$updateBranch/update-algo-system-files/$((Get-Date).ToUniversalTime().ToString(`"yyMMddHHmmss`"))" # e.g. main/update-algo-system-files/210101120000
```

Comment:
IMO Either **update-al-go-system-files** or **update-system-files** the product is not called algo.

Reply:
Fixed.